back/node_watcher/src/tasks/createMotion.ts

```diff
@@ -0,0 +1,165 @@
+// Copyright 2018-2020 @paritytech/nomidot authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+import { ApiPromise } from '@polkadot/api';
+import { GenericCall } from '@polkadot/types';
+import { BlockNumber, Hash } from '@polkadot/types/interfaces';
+import { logger } from '@polkadot/util';
+
+import { prisma } from '../generated/prisma-client';
+import { motionStatus } from '../util/statuses';
+import { NomidotMotion, NomidotMotionRawEvent, Task } from './types';
+
+const l = logger('Task: Motions');
+
+/*
+ *  ======= Table (Motion) ======
+ */
+const createMotion: Task<NomidotMotion[]> = {
```

Comment:
ah wait btw, these are PolkassemblyMotion! can go do a replacement in another PR to rename all the Polkassembly stuff as such though.

Reply:
Sure. tbh I'd rather name that all "NomidotBlabla" because we do share things already such as blockbnumbrer, and the future might add some more..

Reply:
in that case then we should remove Nomidot prefix entirely imo

Reply:
Agree, let's use another PR for that though :smile: